src/containers/CheckoutPage.less

```diff
@@ -0,0 +1,364 @@
+          line-height: 1.428571429;
+          text-align: center;
+          vertical-align: middle;
+          cursor: pointer;
+          border: 1px solid transparent;
+          border-radius: 4px;
+          white-space: nowrap;
+          -webkit-user-select: none;
+          -moz-user-select: none;
+          -ms-user-select: none;
+          -o-user-select: none;
+          user-select: none;
+          border-radius: 2px;
+          -webkit-box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.2);
+          box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.2);
+          margin-bottom: 2px;
+          color: #ffffff;
+          background-color: #040060;
+          border-color: #040060;
+        }
+
+        .btn-cmd-submit-order:focus {
+          outline: thin dotted #333;
+          outline: 5px auto -webkit-focus-ring-color;
+          outline-offset: -2px;
+        }
+
+        .btn-cmd-submit-order:hover,
+        .btn-cmd-submit-order:focus {
+          color: #333333;
+          text-decoration: none;
+        }
+
+        .btn-cmd-submit-order:active,
+        .btn-cmd-submit-order.active {
+          outline: 0;
+          background-image: none;
+          -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+          box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+        }
+
+        .btn-cmd-submit-order.disabled,
+        .btn-cmd-submit-order[disabled],
+        fieldset[disabled] .btn-cmd-submit-order {
+          cursor: not-allowed;
+          pointer-events: none;
+          opacity: 0.65;
+          filter: alpha(opacity=65);
+          -webkit-box-shadow: none;
+          box-shadow: none;
+        }
+
+        .btn-cmd-submit-order:hover,
+        .btn-cmd-submit-order:focus,
+        .btn-cmd-submit-order:active,
+        .btn-cmd-submit-order.active,
+        .open .dropdown-toggle.btn-cmd-submit-order {
+          color: #ffffff;
+          background-color: #040060;
+          border-color: #040060;
+        }
+
+        .btn-cmd-submit-order:active,
+        .btn-cmd-submit-order.active,
+        .open .dropdown-toggle.btn-cmd-submit-order {
+          background-image: none;
+        }
+
+        .btn-cmd-submit-order.disabled,
+        .btn-cmd-submit-order[disabled],
+        fieldset[disabled] .btn-cmd-submit-order,
+        .btn-cmd-submit-order.disabled:hover,
+        .btn-cmd-submit-order[disabled]:hover,
+        fieldset[disabled] .btn-cmd-submit-order:hover,
+        .btn-cmd-submit-order.disabled:focus,
+        .btn-cmd-submit-order[disabled]:focus,
+        fieldset[disabled] .btn-cmd-submit-order:focus,
+        .btn-cmd-submit-order.disabled:active,
+        .btn-cmd-submit-order[disabled]:active,
+        fieldset[disabled] .btn-cmd-submit-order:active,
+        .btn-cmd-submit-order.disabled.active,
+        .btn-cmd-submit-order[disabled].active,
+        fieldset[disabled] .btn-cmd-submit-order.active {
+          background-color: #040060;
+          border-color: #040060;
+        }
```

Comment:
This is a lot of styling for a button. Is it ever disabled? If not we don't need this. If we do need this you probably need just one selector like `.disabled`.